resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -145,6 +146,20 @@ private[spark] object Config extends Logging {
       .checkValue(_ >= 0, "Interval should be non-negative")
       .createWithDefault(0)
 
+  object ExecutorRollPolicy extends Enumeration {
+    val ID, ADD_TIME, TOTAL_GC_TIME, TOTAL_DURATION = Value
+  }
+
+  val EXECUTOR_ROLL_POLICY =
+    ConfigBuilder("spark.kubernetes.executor.rollPolicy")
+      .doc("Executor roll policy: Valid values are ID, ADD_TIME, TOTAL_GC_TIME (default), " +
+        "and TOTAL_DURATION")
```

Comment:
Do we need to describe the meanings for each policy like the description does?

Reply:
Thank you for review. Sure! I'll add.